src/app/communities/communities.service.ts

```diff
@@ -41,4 +41,8 @@ export class CommunitiesService {
     return this.httpClient.post<CommunityResponse>(this.joinCommunityUrlPattern.replace('{communityId}', communityId), null);
   }
 
+  leaveCommunity(communityId: string): Observable<void> {
```

Comment:
There should be `CommunityResponse` instead of `void`.

Reply:
done